engine/common/imagelib/img_png.c

```diff
@@ -94,9 +97,13 @@ qboolean Image_LoadPNG( const char *name, const byte *buffer, fs_offset_t filesi
 		return false;
 	}
 
-	if( png_hdr.ihdr_chunk.colortype != PNG_CT_RGB && png_hdr.ihdr_chunk.colortype != PNG_CT_RGBA )
+	if( !( png_hdr.ihdr_chunk.colortype == PNG_CT_RGB
+	    || png_hdr.ihdr_chunk.colortype == PNG_CT_RGBA
+	    || png_hdr.ihdr_chunk.colortype == PNG_CT_GREY
+	    || png_hdr.ihdr_chunk.colortype == PNG_CT_ALPHA
+	    || png_hdr.ihdr_chunk.colortype == PNG_CT_PALLETE ) )
 	{
-		Con_DPrintf( S_WARN "Image_LoadPNG: Only 8-bit RGB and RGBA images is supported (%s)\n", name );
+		Con_DPrintf( S_WARN "Image_LoadPNG: Unknown color type (%s)\n", name );
```

Comment:
Maybe output the value of `png_hdr.ihdr_chunk.colortype`? It will help troubleshooting in case of broken files or some new unsupported mode.